### ISSUE TITLE
Add movie status to activity tab

### ISSUE THREAD

**Issue by lunks** [Radarr/Radarr#4501]
**Is your feature request related to a problem? Please describe.**
Whenever I go to the activity section and I see a download with an issue, I am not sure if the file is for a movie that is missing, if it is a newer release or not. I have to go into each movie to check its status.

**Describe the solution you'd like**
Allow me to see the movie status, if it is missing, downloaded or not. I can opt to just remove a release for a movie that has been downloaded previously. Ideally it'd be nice to see the tags for the current movie as well, but just movie status would be a lifesaver.

**Describe alternatives you've considered**
Go into the movie and check its status individually.

**Comment by austinwbest** [comment]
If you mouseover the icon (to the left of the movie) it should tell you the status

**Comment by lunks** [comment]
Hey, @austinwbest. I just upgraded to v3 to check it out. I started grabbing a new movie and I can't see a "Missing" status which is what is shown on the movie page. I'm talking about the movie status, not the download file status.

**Comment by austinwbest** [comment]
You're going to see a download status in the activity section typically as that is kinda what that's for. There is 1 status in the UI (discover, movie details, queue, etc) with a dozen or so possible options based on the state of the movie

**Comment by lunks** [comment]
The scenario I commonly have is: A file is downloaded but I may or may not have downloaded it before. I have to individually go into the problematic movie page to check whether it has been previously downloaded or if it is missing. I'd like to skip this step by seeing it somehow on the same page.

**Comment by austinwbest** [comment]
Ok, so maybe i am overthinking or something as i still dont see it..

Activity tab is a place of what is currently going on with a specific download. If there is an existing one or not plays no part in it. 

If it is downloaded previously (upgrade now) or new (missing) doesnt matter for the specific file in the activity. The error still has to be resolved so i'm not sure how that will help/matter. Do you throw it out if it exists already (an upgrade) or something like that and this is why you bounce back and forth?

At most showing if it is an upgrade/new on this would be something to consider (just for the information).

**Comment by lunks** [comment]
To provide a more concrete use-case: Sometimes a download is listed as Bluray-1080p and the file is detected as WebDL-1080p. Radarr stops the process and I can either manually import it or delete it.

But now I have to consider: 

Has it been downloaded already? If it is, I now have to decide whether it's worth the upgrade or not and manually import it.

If it has not been dowloaded previously, I don't care if it's a WebDL reported as Bluray, I don't have the movie yet. I'd like to import it.

Sometimes I have a big chunk of movies with a bunch of them like that. I have to manually go into each movie to check if the movie has been downloaded previously or not to proceed. So I open each movie into a new tab, check them individually and decide on them individually. Having a way to check if the movie has been downloaded or not on the activity tab would be of big help.

I could go further and say that having the filename and tags for the already downloaded file, which are present on the movie tab, would also help, but I can see a point being made that these don't really belong to the activity tab. But having just the movie status (that is, Downloaded/Missing) would already be a quality of life improvement I'd appreciate greatly.

**Comment by austinwbest** [comment]
> Has it been downloaded already? If it is, I now have to decide whether it's worth the upgrade or not and manually import it.
> 
> If it has not been dowloaded previously, I don't care if it's a WebDL reported as Bluray, I don't have the movie yet. I'd like to import it.

That helped.. There is a lot of information already available so i would say at most it would be something somewhere that denotes if it is new or an upgrade. 

To go more granular than that leads to a problem of being many reasons for an upgrade and which one it upgraded for might not be the one you want to see on that page. So the amount of choices becomes large.

**Comment by lunks** [comment]
If I understand it correctly in the case the movie is missing it would be
shown as “new”? If it is, that’d solve my use case.

**Comment by austinwbest** [comment]
Yes, if there are no files on disk (missing for whichever status) then it would shows as "New" else "Upgrade"

**Comment by dramirvf** [comment]
I believe including this feature would be very useful.
Additionally, the column including the information on these information are not currently sortable, which is very annoying and I have described it here:
https://github.com/Radarr/Radarr/issues/5252

**Comment by austinwbest** [comment]
You realize you don't need to promote your bug on other feature requests right? We see your ticket bud, we will review it, reproduce it, fix what needs fixing :)

As for this request, some firm of implementation is on my list. Wrapping up a few other things that are a little higher priority first is all

**Comment by dramirvf** [comment]
> You realize you don't need to promote your bug on other feature requests right? We see your ticket bud, we will review it, reproduce it, fix what needs fixing :)
> 
> As for this request, some firm of implementation is on my list. Wrapping up a few other things that are a little higher priority first is all

Some issues and features are relevant to each other and require simultaneous considerations.

The issues in these 2 posts are both relevant to "Status" column and addressing them separately may need extra job to fix similar issue.

I think in the first place, the "Status Column" has not been defined consistently across Radarr. Somewhere it refers to "Release Status" and somewhere it refers to "Availability Status".

This discrepancy should be addressed first. If it is solved then you'll have a separate value for availability which you could name it different from "Status", for example "Availability", which includes values such as "Downloaded, Missing, Downloading, etc". Now, here it is possible to add "Upgrading" value as well, which is the main idea of the current post. 

Then "Availability" could be used across radarr. For example as a filter in "Custom Filters", as a column in "Activity" tab, as a column in "Movie Table View", and as an overlay text in "Movie Poster View".

**Comment by geogolem** [comment]
> > You realize you don't need to promote your bug on other feature requests right? We see your ticket bud, we will review it, reproduce it, fix what needs fixing :)
> > As for this request, some firm of implementation is on my list. Wrapping up a few other things that are a little higher priority first is all
> 
> Some issues and features are relevant to each other and require simultaneous considerations.
> 
> The issues in these 2 posts are both relevant to **"Status"** column in **"Activity Tab"** and addressing them separately may need extra job to fix similar upcoming issues.
> 
> I think in the first place, the **"Status"** parameter has not been defined consistently across Radarr. Somewhere it refers to **"Release Status"** and somewhere it refers to **"Availability Status"**.
> 
> This discrepancy should be addressed first. If it is solved then you'll have a separate parameter for availability which you could name it different from **"Status"**, for example **"Availability"**, which includes values such as **"Current Downloaded Quality, Missing, Downloading, Error, Paused, etc"**. Now, here it is possible to add **"Upgrading"** value as well, which is the main idea of the current post.
> 
> Then, **"Availability"** parameter could be used across Radarr. For example as a filter in **"Custom Filters"**, as a column in **"Activity"** tab, as a column in **"Movie Table View"**, and as an overlay text in **"Movie Poster View"**.

The issue regarding custom filters and "Status" column has been addressed in an existing PR: 
https://github.com/Radarr/Radarr/pull/4924

There is also further discussion about this here: https://github.com/Radarr/Radarr/issues/5272
